deploy/base/clusterrole.yaml

```diff
@@ -229,5 +229,13 @@ rules:
     - authentication.k8s.io
   resources:
     - tokenrequests
+    - tokenreviews # used in agent-registration's authentication
   verbs:
     - create
+- apiGroups: # used in agent-registration's autherization
+    - "authorization.k8s.io"
+  resources:
+    - subjectaccessreviews
+  verbs:
+    - create
+    - get
```

Comment:
seems there is no get for subjectaccessreviews?

Reply:
Good point, remove "get".